### ISSUE TITLE
/debug is completely broken!

### ISSUE THREAD

**Issue by juanmuscaria** [CrucibleMC/Crucible#4]
The internal profiler is apparently broken, if you type /debug start it will cause a high cpu usage and a memory leak.
```
23.12 13:21:09 [Server] WARN Something's taking too long! 'root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.levels.world.tick' took aprox 287.903663 ms
23.12 13:21:09 [Server] WARN Something's taking too long! 'root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.levels.world' took aprox 288.161519 ms
23.12 13:21:09 [Server] WARN Something's taking too long! 'root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.root.levels' took aprox 289.413707 ms
```
We have two options, completely remove the minecraft internal profiler and replace it with timings or fix the internal profiler.

**Comment by evernife** [comment]
I think this doe not work since before thermos.
No one uses.
Better remove it.

**Comment by juanmuscaria** [comment]
a739f11d8e9798f5109fb72d7715d910a85fdcdf

**Comment by spannerman79** [comment]
This was broken/borked since MCPC+.  Remove it and keep it removed.  From memory bloodmc was planning on removing it or changing it to use a warmroast type of report instead.

**Comment by evernife** [comment]
With timings v2 implemented, and having opis to be used on 1.7.10 servers as well, i dont think this needs to be fixed at all :D

**Comment by juanmuscaria** [comment]
I will be closing this issue as it is possible to use timings and spark to do the same thing that debug did.
